core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.html

```diff
@@ -10,29 +10,34 @@
     <div class="col-md-3 oppia-learner-dashboard-side-content"
          *ngIf="!windowIsNarrow">
       <mat-card class="oppia-learner-dashboard-main-menu">
-        <h4 class="oppia-learner-dashboard-menu oppia-learner-dashborad-menu-hover protractor-test-home-section"
+        <h4 class="oppia-learner-dashboard-menu oppia-learner-dashborad-menu-hover protractor-test-home-section" tabindex="0"
             [ngClass]="{'oppia-learner-dashboard-section-active': activeSection === LEARNER_DASHBOARD_SECTION_I18N_IDS.HOME}"
-            (click)="setActiveSection(LEARNER_DASHBOARD_SECTION_I18N_IDS.HOME)">
+            (click)="setActiveSection(LEARNER_DASHBOARD_SECTION_I18N_IDS.HOME)"
+            (keyup)="setActiveTab($event, LEARNER_DASHBOARD_SECTION_I18N_IDS.HOME)">
```

Comment:
Can we bind enter key to the key event here? e.g. https://angular.io/guide/user-input#key-event-filtering-with-keyenter

That way we can simply call setActiveSection.

Reply:
Thanks, this makes it much easier. Fixed.